.github/workflows/image_rs_build.yml

```diff
@@ -96,6 +96,10 @@ jobs:
       # - name: Run cargo test - kata-cc (native-tls version) with keywrap-grpc + keywrap-jwe
       #   run: |
       #     sudo -E PATH=$PATH -s cargo test -p image-rs --no-default-features --features=encryption-openssl,keywrap-grpc,snapshot-overlayfs,signature-cosign-native,signature-simple,getresource,oci-distribution/native-tls,keywrap-jwe
+      
+      - name: Run cargo test - nydus
+        run: |
+          sudo -E PATH=$PATH -s cargo test -p image-rs --features nydus
```

Comment:
Do you need sudo here?

Reply:
This is consistent with the other test commands.

Reply:
the mount operation in test cases need the root privilege